### PR TITLE
Create Candu-cohort.md

### PR DESCRIPTION
**Pull request by Brandoncharleskhoo1**
# Amplitude Developer Docs PR

## Description

Describe your changes. 

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/destinations/candu-cohort.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+- You must enable this integration in each Amplitude project you want to use it in.
+
+## Setup
+
+To configure the Cohort integration from Amplitude to Candu, complete the following steps in each platform.
+
+### Candu 
+
+In Candu, navigate to **Settings** then click **Integrations**.
```
Comment (markzegarelli) [inline docs/data/destinations/candu-cohort.md:22]: @Brandoncharleskhoo1 What else must happen in Candu? Users simply need to go to an Integrations page?

Comment (Brandoncharleskhoo1) [reply docs/data/destinations/candu-cohort.md]: Updated the steps! Good catch!

**Comment by Brandoncharleskhoo1** [comment]
Screenshot 2024-05-01 at 11 21 44 PM

@markzegarelli I don't why it's failing :'(. It's referencing the right doc candu-cohort.md

**Comment by markzegarelli** [comment]
@Brandoncharleskhoo1 It didn't think `Candu` is a word. I added it to the accept list, and it should be good to go now.

**Review by markzegarelli — APPROVED** [review]

**Pull request merged**